test/cpp/jit/test_gpu.cpp

```diff
@@ -6836,14 +6990,12 @@ TEST(NVFuserTest, FusionMagicSchedulerLayerNormalization_CUDA) {
 
   std::vector<int> reduction_axes(norm_shape.size());
   std::vector<bool> broadcast_mask(input->nDims(), false);
-  Val* num_features = nullptr;
+  Val* num_features = new Double(1);
```

Comment:
nitpick: maybe `Int`?

Reply:
The result type for `reciprocal(x)` and `div(1, x)` is the type of the argument `x`.
If it is `Int`, the output will be an `Int' and the result will be 0. 
Also, we have 2 versions of reciprocal for double and float, so it complains about which one to select.